pkg/tests/tasks/traffic/common.go

```diff
@@ -0,0 +1,62 @@
+package traffic
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/maistra/maistra-test-tool/pkg/util/oc"
+	"github.com/maistra/maistra-test-tool/pkg/util/pod"
+	"github.com/maistra/maistra-test-tool/pkg/util/template"
+
+	. "github.com/maistra/maistra-test-tool/pkg/util/test"
+)
+
+func TestreviewV1(t TestHelper, file string) string {
+
+	ns := "bookinfo"
+
+	reviewV1Podname := pod.MatchingSelector("app=reviews,version=v1", ns)(t, oc.DefaultOC).Name
+
+	templateString, err := os.ReadFile("../../../../testdata/resources/html/" + file)
```

Comment:
```suggestion
	templateString, err := os.ReadFile(env.GetRootDir() + "/testdata/resources/html/" + file)
```
I found we can use `env.GetRootDir()` to get the root directory directly (without needing all the "../"). You'll need to import `env` first.